### ISSUE TITLE
Easier to use with RunwayML Beta?

### ISSUE THREAD

**Issue by niklaskeerl** [runwayml/unity#6]
I would like to use the "Face-Recognition" model in Unity and deploy it to the HoloLens. Do you think this could work with the beta?

**Comment by cvalenzuela** [comment]
Yes! You could send data frames between Runway and the HoloLens with a simple HTTP request. Take a look here: https://docs.runwayml.com/#/how-to/network

**Comment by niklaskeerl** [comment]
Thank you for the suggestion. Now, all I need to focus on is getting the picture from the Unity Camera and send the HTTP POST request as json with the base 64 encoded string of the picture.
I will keep this issue updated when I get to work on this project if anyone is interested.
You can close this issue if you want to.
